include/fizzy/fizzy.h

```diff
@@ -206,6 +232,18 @@ uint32_t fizzy_get_global_count(const FizzyModule* module);
 /// @note  All module global indices are greater than all imported global indices.
 FizzyGlobalType fizzy_get_global_type(const FizzyModule* module, uint32_t global_idx);
 
+/// Get number of imports defined in the module.
+///
+/// @param  module    Pointer to module. Cannot be NULL.
+uint32_t fizzy_get_import_count(const FizzyModule* module);
+
+/// Get the import type defined in the module.
+///
+/// @param  module        Pointer to module. Cannot be NULL.
+/// @param  import_idx    Import index. Behaviour is undefined, if index is not valid according
+///                       to module definition.
```

Comment:
Missing `@returns`. I would also explain the lifetime of `FizzyImport.module` and `FizzyImport.name`.